### PR TITLE
No terrorism

### PR DESCRIPTION
**Pull request by Qrox**
#### Summary
SUMMARY: Bugfixes "Fix an incorrect philosophy book title"

#### Purpose of change
[Leon Trotsky](https://en.wikipedia.org/wiki/Leon_Trotsky) never wrote [anything defending terrorism](https://www.marxists.org/archive/trotsky/idom/dm/index.htm). Correct me if I'm wrong.

#### Describe the solution
Correct the book title.

#### Describe alternatives you've considered
If it was kind of a joke, then it might offend someone.

### CODE REVIEW

**Comment by roaringjohn** [comment]
https://www.forgottenbooks.com/en/books/TheDefenceofTerrorism_10045249

**Comment by Qrox** [comment]
Huh, didn't notice that when I searched, thanks. Anyway, it should be corrected to either of these two titles, not a mix of the two.

File: data/json/items/book/misc.json [hunk 1]
```diff
@@ -1021,7 +1021,8 @@
       {
         "id": "philosophy30",
         "text": "This is a copy of both \"On The Genealogy of Morals\" and \"The Gay Science\" by Friederich Nietzsche."
-      }
+      },
+      { "id": "philosophy31", "text": "This is a copy of \"The Defence of Terrorism\" by Leon Trotsky.  Despite the title, it does not actually appear to be defending terrorism." }
```
Comment (kevingranade) [inline data/json/items/book/misc.json]:
```suggestion
      {
        "id": "philosophy31",
        "text": "This is a copy of \"The Defence of Terrorism\" by Leon Trotsky.  Despite the title, it does not actually appear to be defending terrorism."
      }
```

**Comment by KPuJu** [comment]
> https://www.forgottenbooks.com/en/books/TheDefenceofTerrorism_10045249

Maybe its late but its called "Terrorism and Communism"
https://ru.wikipedia.org/wiki/%D0%A2%D0%B5%D1%80%D1%80%D0%BE%D1%80%D0%B8%D0%B7%D0%BC_%D0%B8_%D0%BA%D0%BE%D0%BC%D0%BC%D1%83%D0%BD%D0%B8%D0%B7%D0%BC_(%D0%A2%D1%80%D0%BE%D1%86%D0%BA%D0%B8%D0%B9)

**Comment by Mister-Cultist** [comment]
defending communism is worse than defending terrorism, but hey it's not terrorism :^)

**Comment by roaringjohn** [comment]
> > https://www.forgottenbooks.com/en/books/TheDefenceofTerrorism_10045249
> 
> Maybe its late but its called "Terrorism and Communism"
> https://ru.wikipedia.org/wiki/%D0%A2%D0%B5%D1%80%D1%80%D0%BE%D1%80%D0%B8%D0%B7%D0%BC_%D0%B8_%D0%BA%D0%BE%D0%BC%D0%BC%D1%83%D0%BD%D0%B8%D0%B7%D0%BC_(%D0%A2%D1%80%D0%BE%D1%86%D0%BA%D0%B8%D0%B9)

Indeed, however...

> Trotsky's book, the first English edition of which bore the title The Defense of Terrorism...

**Comment by LaVeyanFiend** [comment]
https://www.reddit.com/r/cataclysmdda/comments/cty2qf/someone_already_made_the_joke_but_i_spent_2/
image

**Comment by faefux** [comment]
I'm really confused about this PR.  Has it been changed to 'In defence of Communism'?  It's not a huge problem but that's 100% not what was being referenced here.   Can i revert this?

**Comment by Qrox** [comment]
Both "In defence of Communism" and "The Defence of Terrorism" were added, because these are what we could find that are closest to the original reference. If you can provide some proof that Trotsky actually wrote a book named "In Defence of Terrorism", I'm perfectly fine to revert this change.

> I'm uncomfortable with this logic being used.
The logic was that there wasn't a book by Trotsky published under the exact title "In Defence of Terrorism". It being a joke was just an unlikely possibility I was also considering.

**Pull request merged**